package/etc/php.ini

```diff
@@ -0,0 +1,1870 @@
+﻿[PHP]
```

Comment:
Shouldn't we ship the prod and dev version? Couldn't we copy them from
the PHP source tree at build time instead of copying them in our Git repo?

Reply:
Yes, good idea. I'll have to remember to change the Dockerfile to remove sources after building packages.